tests/testBackground.py

```diff
@@ -113,9 +89,9 @@ def testOddSize(self):
 
         for iy, by in zip([image.get(0, y) for y in range(H)],
                           [back.get(0, y) for y in range(H)]):
-            self.assertTrue(abs(iy - by) < 5)
+            self.assertLess(abs(iy - by), 5)
 
-        if False:
+        if display:
```

Comment:
I'm happy to just delete this matplotlib block entirely. but I figured that it should be wrapped in something accessible from the outside.